deploy/kubernetes/kubectl/wsnext/feature-api.yaml

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
It would be nice to have few words in `Deploy Che.md` about deploying `feature-api`

Reply:
ok